test/e2e/framework.go

```diff
@@ -877,7 +892,23 @@ func (data *TestData) deployFlowAggregator(ipfixCollector string) error {
 	if err != nil || rc != 0 {
 		return fmt.Errorf("error when deploying the Flow Aggregator; %s not available on the control-plane Node", flowAggYaml)
 	}
-	if err = data.mutateFlowAggregatorConfigMap(ipfixCollector); err != nil {
+	// clickhouse-ca Secret is created in the flow-visibility namespace. In order to make it accessible for Flow Aggregator,
```

Comment:
capitalize all occurrences of "Namespace" in this comment

Reply:
Fixed.